docs/ibexa_engage/install_ibexa_engage.md

```diff
@@ -0,0 +1,53 @@
+---
+description: Install and configure Ibexa Engage.
+edition: experience
+---
+
+# [[= product_name_engage =]]
+
+[[= product_name_engage =]] is a data collection tool. It enables you to engage your audiences by using the [Qualifio](https://qualifio.com/) tools.
+You can use interactive content to gather valuable data, for example, customer data or recent orders list, and create connections.
+
+For more information, see [Qualifio Developers documentation](https://developers.qualifio.com/docs/engage/).
+
+## Enable [[= product_name_engage =]] account
+
+To use [[= product_name_engage =]], you must make arrangements with [[= product_name =]] to define the initial configuration.
+[[= product_name_base =]] team creates and provides user account. An invitation link is sent during the setup process.
+
+For more information, see [Ibexa Engage in User documentation]([[= user_doc =]]/ibexa_engage/ibexa_engage/#request-access).
+
+## Install [[= product_name_engage =]]
+
+[[= product_name_engage =]] comes from v4.6.6 of [[= product_name_exp=]].
+If you have different version, run the following command to install the bundle:
+
+``` bash
+composer require ibexa/engage
+```
+
+You can check for its presence by using the following command:
+
+``` bash
+composer show | grep "ibexa/engage"
+```
+
+This command adds to your project configuration files required for using [[= product_name_engage =]].
+
+## Prepare configuration files
+
+In `config/packages` directory add the following `ibexa_connector_qualifio` [YAML configuration](configuration.md#configuration-files):
```

Comment:
```suggestion
In `config/packages` directory add the following `ibexa_connector_qualifio.yaml` [YAML configuration](configuration.md#configuration-files):
```

It's better to mention the full filename, with extension - less chance for people to make a mistake.

Or to improve the DX even further - @Steveb-p could we create a default configuration file using recipes? I see https://github.com/ibexa/recipes-dev/tree/master/ibexa/connector-qualifio/4.6 adds the routing, maybe it could create the configuration as well?

We're doing something like this for Actito: https://github.com/ibexa/recipes-dev/blob/master/ibexa/connector-actito/4.6/config/packages/ibexa_connector_actito.yaml